SimG4CMS/Forward/src/TimingSD.cc

```diff
@@ -201,7 +201,7 @@ bool TimingSD::hitExists(const G4Step* aStep) {
   //tSliceID already exists:
 
   bool found = false;
-  for (int j = 0; j < theHC->entries(); ++j) {
+  for (size_t j = 0; j < theHC->entries(); ++j) {
```

Comment:
@mrodozov , this is fine - only one loop, so switch to size_t does not change anything.